v22.1/backup.md

```diff
@@ -140,7 +140,25 @@ This improves performance by decreasing the likelihood that the `BACKUP` will be
 
 {% include_cached new-in.html version="v22.1" %} A backup job will [pause](pause-job.html) instead of entering a `failed` state if it continues to encounter transient errors once it has retried a maximum number of times. Once the backup has paused, you can either [resume](resume-job.html) or [cancel](cancel-job.html) it.
 
-{% include {{ page.version.version }}/backups/file-size-setting.md %}
+### Backup performance configuration
+
+Cluster settings provide a means to tune a CockroachDB cluster. The following cluster settings are helpful for configuring backup files and performance:
+
+#### `bulkio.backup.file_size` 
+
+Set a target for the amount of backup data written to each backup file. This is the maximum target size the backup will reach, but it is possible files of a smaller size are created during the backup job.
+
+If `bulkio.backup.file_size` is set too low, it will cause the backup job to create many small SST files, which could impact a restore job's performance because it will need to keep track of so many small files.
```

Comment:
Question: do we have an idea of what constitutes "too low"? Guess: "too low for your deployment's average SST size", but that's difficult to both advise of and track on the end user's side. If there's no guidance from engineering on the value of this tunable, ignore me and leave this as is.

Reply:
I have adapted this after discussing with Aditya. We want to refer to the lowering from the default here, which I have changed. TFTR!